code/modules/clothing/glasses/_glasses.dm

```diff
@@ -27,6 +27,11 @@
 	if(glass_colour_type && ishuman(user))
 		. += "<span class='notice'>Alt-click to toggle its colors.</span>"
 
+/obj/item/clothing/glasses/proc/prescribe()
+	vision_correction = TRUE
+	name = "prescription [src]"
```

Comment:
i would do "prescription [initial(name)]" here

Reply:
i'm just slapping "prescription" on the name because of the nonzero chance someone might have snowflake non-prescription glasses they want to prescription up

Reply:
yeah i can see that, it would probably just be better to use initial(name) to get the name here instead of src

Reply:
think if it was initial(name) it'd get the base name? could do src.name or something idk